src/test/javascript/cypress/integration/entity/outcome.spec.ts

```diff
@@ -110,29 +110,29 @@ describe('Outcome e2e test', () => {
     cy.visit('/');
   });
 
-  it('should delete last instance of Outcome', () => {
```

Comment:
Please assure that we aren't deleting an automatic test necessary to the "view list" functionality